addons/Dimmer/Dimmer.lua

```diff
@@ -38,6 +38,7 @@ item_info = {
     [1]={id=26176,japanese='Ｄ．ホラリング',english='"Dim. Ring (Holla)"',slot=13},
     [2]={id=26177,japanese='Ｄ．デムリング',english='"Dim. Ring (Dem)"',slot=13},
     [3]={id=26178,japanese='Ｄ．メアリング',english='"Dim. Ring (Mea)"',slot=13},
+    [4]={id=10385,japanese="キュムラスマスク+1",english="Cumulus Masque +1",slot=4},
 	}
```

Comment:
Tab D:<

Reply:
Pretty sure that was already there! (fix'd)